man/subscription-manager.8

```diff
@@ -250,9 +250,15 @@ Sets the preferred service level to use with subscriptions added to the system.
 
 .TP
 .B --force
-Registers the system even if it is already registered. Normally, any register operations will fail if the system is already registered. With
+Normally, register operations will fail if the system is already registered.  With
 .B --force,
-the existing system entry is unregistered first, all of its subscriptions are returned to the pool, and then the system is registered as a new entry.
+the existing system identity is unregistered first, all of its attached subscriptions are returned to their pools, and then the system is registered with a new identity.   Beware that the
+.B --force
+option does not guarantee a successful registration.  For example, if the registration with
+.B --force
+includes a different
+.B --serverurl
+than was used for the original registration, the implicit call to unregister from the original entitlement server will fail with invalid credentials and the registration with force will be aborted.  In this case, the user should explicity unregister from the original entitlement server.  If unregistering is not possible, then running subscription-manager clean will effectively abandon the original registration identity and entitlements.  Once cleaned, registering a new system identity should succeed with or without force.
```

Comment:
missing "that" between "than" and "was used", perhaps?

Reply:
I think this is OK